src/awfy.js

```diff
@@ -519,6 +519,11 @@ export const CONFIG = {
       platforms: ['macosx1014-64-shippable-qr'],
       categories: DESKTOP_CATEGORIES,
     },
+    mac1400: {
+      label: 'macOS 14.5 "Sonoma"',
```

Comment:
Thoughts on keeping this label? I anticipate we will keep upgrading the M2 OS versions like we have done so far, so the Sonoma label will get outdated eventually. Should we just be more generic Mac OS 14.x (but that could also become 15.x in the future) or just leave as is here in the PR, and update as required, with the understanding that when someone looks at the data it encompasses different macOS versions depending on the date?

Reply:
I have no strong preference